lib/node_modules/@stdlib/array/base/to-inserted-at/lib/main.js

```diff
@@ -0,0 +1,68 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var normalizeIndex = require( '@stdlib/ndarray/base/normalize-index' );
+var zeros = require( '@stdlib/array/zeros' );
+var dtype = require( '@stdlib/array/dtype' );
+var format = require( '@stdlib/string/format' );
+var assign = require( './assign.js' );
+
+
+// MAIN //
+
+/**
+* Returns a new array containing every element from an input array, with a provided value inserted at a specified index.
+*
+* @param {Collection} x - input array
+* @param {integer} index - element index
+* @param {*} value - value to insert
+* @throws {RangeError} second argument must not exceed array bounds
+* @returns {Collection} output array
+*
+* @example
+* var x = [ 1, 2, 3, 4 ];
+*
+* var v = toInsertedAt( x, 0, 5 );
+* // returns [ 5, 1, 2, 3, 4 ]
+*
+* v = toInsertedAt( x, 1, 6 );
+* // returns [ 1, 6, 2, 3, 4 ]
+*
+* v = toInsertedAt( x, -2, 7 );
+* // returns  [ 1, 2, 7, 3, 4 ]
+*/
+function toInsertedAt( x, index, value ) {
+	var out;
+
+	index = normalizeIndex( index, x.length-1 );
```

Comment:
```suggestion
	index = normalizeIndex( index, x.length );
```

In order to ensure we can support both prepend and append ops, we likely want to bump this by one. Otherwise, append is not allowed.